### DISCUSSION TITLE
VALIDATION RESULTSがWAITINGから更新されません

### DISCUSSION THREAD

**Discussion by kanosan-wataru** [smly/mjai.app#88] [Bug report] [answered]
2週間ほどWAITINGから更新されていない状態です。
ご確認をよろしくお願いいたします。
![image](https://github.com/smly/mjai.app/assets/40311715/7c2913a8-bd0f-497c-aa83-4a7d559280b2)

**Comment by Nitasurin** [comment]
@smly 
I also encountered this problem.

**Comment by smly** [comment]
@karon9 @Niitsune Thank you for reporting this! It may be a problem with the validation server. I will check it now. :pray:

**Reply by smly** [reply]
I have checked the validation server logs. I found that one of the submissions failed to upload. As a result, only some user submissions, including test cases, were validated, and some user submissions were not validated.

I fixed the problem by editing the database directly. The validation server is being started now.
Please wait a moment while validation is completed. I will implement Slack notifications when errors occur.

**Reply by smly** [reply, marked as answer]
@karon9 @Niitsune OK! The problem has been resolved. All stacked submissions are now evaluated by the validation server.

**Reply by Nitasurin** [reply]
Thank you for fixing it so quickly! I correct the errors in my submission and it has been validated.

**Reply by kanosan-wataru** [reply]
対応していただきありがとうございました！
